neuro_py/tuning/fields.py

```diff
@@ -91,7 +88,7 @@ def find_peaks(image: np.ndarray) -> np.ndarray:
     """
     image = image.copy()
     image[~np.isfinite(image)] = 0
-    image_max = filters.maximum_filter(image, 3)
+    image_max = ndimage.maximum_filter(image, 3)
```

Comment:
I feel like the 3 here should be configurable instead of hard-coded. Maybe default argument the function itself